### ISSUE TITLE
Do not show top languages

### ISSUE THREAD

**Issue by Giorgi** [piotrl/github-profile-card#25]
Looks like most of the time for getting the data is spent on getting language details so can we have an option to skip language details and don't show it?

**Comment by piotrl** [comment]
Looks like good idea to optionally disable this functionality.

Would you like to jump into this maybe? 
Here's section of code that'd need to be disabled:
* https://github.com/piotrl/github-profile-card/blob/master/src/gh-profile-card.ts#L96

New configuration would need to be handled:
* HTML config: https://github.com/piotrl/github-profile-card/blob/master/src/gh-profile-card.ts#L55
* default programmable config: https://github.com/piotrl/github-profile-card/blob/master/src/gh-profile-card.ts#L32

And here is small app to test:
* https://github.com/piotrl/github-profile-card/blob/master/src/gh-profile-card.ts#L32

Dev process:
* `yarn install`
* `yarn dev` -> https://github.com/piotrl/github-profile-card/blob/master/package.json#L25

**Comment by Giorgi** [comment]
When I run `yarn dev` I get:

```
'WEBPACK_ENV' is not recognized as an internal or external command, operable program or batch file.
```

Any suggestions ?

**Comment by piotrl** [comment]
are you using windows cmd?

Maybe you can try git-bash that imitates linux-like CLI? https://gitforwindows.org/

**Comment by Giorgi** [comment]
Same error with git-bash

**Comment by Giorgi** [comment]
Solved it with cross-env

**Comment by piotrl** [comment]
Super cool, thanks @Giorgi for contribution! This is so awesome, you're first contributor of this project :D 

I released version 3.1.0 with some fixes. See it here:
https://github.com/piotrl/github-profile-card/releases

**Comment by Giorgi** [comment]
You are welcome. Thanks for the new release!
